tests/e2e/cli/test_cli.py

```diff
@@ -318,8 +318,8 @@ def test_otx_explain_e2e_cli(
         (p for p in outputs_dir.iterdir() if p.is_dir() and p.name != ".latest"),
         key=lambda p: p.stat().st_mtime,
     )
-    assert (latest_dir / "saliency_maps").exists()
```

Comment:
Why is this updated in this PR? I'm just wondering which components affect this directory name in this PR.

Reply:
It was my attempt to fix broken e2e after https://github.com/openvinotoolkit/training_extensions/pull/3344, I'll revert it